### PR TITLE
add Infinidat CSI driver

### PR DESCRIPTION
**Pull request by gregnsk**
```release-note
NONE
```

### CODE REVIEW

**Comment by xing-yang** [comment]
@gregnsk, the change looks good to me, but you need to sign CLA.

File: book/src/drivers.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [IBM Block Storage](https://github.com/ibm/ibm-block-csi-driver) | `block.csi.ibm.com` | v1.0, v1.1 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/SSRQ8T) for IBM Spectrum Virtualize, IBM Storwize, IBM FlashSystem 9100, A9000 and A9000R, IBM DS8880 and DS8900. | Persistent | Read/Write Single Pod | Yes | Raw Block
 [IBM Spectrum Scale](https://github.com/IBM/ibm-spectrum-scale-csi) | `spectrumscale.csi.ibm.com` | v1.0, v1.1 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/STXKQY_CSI_SHR) for the IBM Spectrum Scale File System | Persistent | Read/Write Multiple Pod | Yes |
+[Infinidat](https://github.com/Infinidat/infinibox-csi-driver) | `infinibox-csi-driver` | v1.0, v1.1 | A Container Storage Interface (CSI) Driver for Infinidat [InfiniBox](https://infinidat.com/en/products-technology/infinibox) | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
```
Comment (msau42) [inline book/src/drivers.md:46]: The repo mentions a user guide. Is that available publicly, or does it come with the product?

Comment (gregnsk) [reply book/src/drivers.md:46]: CLA signed. /assign @xing-yang

Comment (gregnsk) [reply book/src/drivers.md:46]: @msau42 - it's available on Infinidat support site for all InfiniBox customers

**Comment by gregnsk** [comment]
> @gregnsk, the change looks good to me, but you need to sign CLA.

CLA signed. /assign @xing-yang

**Comment by xing-yang** [comment]
@gregnsk It still says "needs CLA".

**Comment by gregnsk** [comment]
I signed it

**Comment by gregnsk** [comment]
I signed it

**Comment by gregnsk** [comment]
Looks like it's signed now
 /assign @xing-yang

**Pull request merged**